app/models/relationship.rb

```diff
@@ -622,6 +605,10 @@ def set_currency_default
       self.currency = 'usd'
     end
   end
+
+  private def refresh_links
```

Comment:
I think you can remove this function and just use `Link.refresh`

Reply:
Sure. Done. Just a personal stylistic habit.